### PR TITLE
Loss callables

### PR DESCRIPTION
**Pull request by t-kalinowski**
- All the loss functions gain the ability to return a callable (a `keras$losses$Loss` instance) if `y_true` and `y_pred` arguments are missing.
- new builtin loss functions: `loss_huber()`, `loss_kl_divergence()`

### CODE REVIEW

**Review by dfalbel — APPROVED** [review]
That's awesome!!
